lib/rucio/common/pcache.py

```diff
@@ -1195,8 +1195,8 @@ def unlock_all(self) -> None:
             try:
                 f.close()
                 os.unlink(filename)
-            except:
-                pass
+            except Exception as e:
+                self.log(INFO, f"Failed to close and unlink {filename}, {e}")
```

Comment:
I'm not sure if using f-string in logging is ideal, for optimization reasons. @dchristidis do you have any thoughts on this?

Reply:
filename and e are already strings, and unless I'm mistaken, aren't f-strings just faster than other options? Not sure what you mean by optimization reasons.

Reply:
Some linters will warn about this: https://pylint.pycqa.org/en/latest/user_guide/messages/warning/logging-fstring-interpolation.html

If the interpolation happens outside the function call, then it happens every time. If it happens inside the function call, then it happens only when necessary. So if your default level is `INFO` and this is a `DEBUG` message, then the interpolation is avoided.

Honestly, I don’t think that it makes a measurable difference performance-wise in our case, but it sounds like a good practice to follow.

Reply:
I opened an issue to enable this linting feature: https://github.com/rucio/rucio/issues/6826